bounties/level-2/federation-server.md

```diff
@@ -0,0 +1,109 @@
+# Implement a federation server for stellar addresses
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 3200 XLM | 1500 XLM | Capped (2) 
+
+[//]: # (make sure to replace the file-name placeholders `<BOUNTY_FILE_NAME_NO_EXTENSION>`, `<BOUNTY_FILE_NAME_WITH_EXTENSION> and `<LEVEL>` in the next two lines with the respective values)
+📜&nbsp; View [existing submissions](https://github.com/tyvdh/stellar-quest-bounties/issues?q=is%3Aissue+label%3Afederation-server) for this bounty. \
+🔵&nbsp; Start [hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.yml&link=https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/federation-server.md) this bounty.
+
+
+## Description
+
+We all love our 56-char stellar account-ids. But what about our friends we want to convince to also join stellar-familiy?
+There are a few arguments could stand in the way of adoption:
+
+- Most of the wallets don't have address-book functionality
+- It is mentally challenging to match a stellar-address to an individual ([vanity-addresses](https://lumenaddr.com/faq.html) help a bit – leaving you with only around 50 chars of gobble)
+
+> "Why can't I send you the money via email?"  
+> -— <cite>Your friends</cite>
+
+Well actually they kind of can; enter [federation].
+
+While there are already numerous federation-servers out there – even a [reference implementation] –
+you might not want to trust an external entity with your and your friends' addresses.
+Further, you might want to hook the addresses to an existing community (think discord, github, twitter, …).
+
+
+### What is this task?
+
+Implement a [federation]-server that allows to sign-up (register) with an existing social-media-account provider
+(e.g. discord, github, google, twitter – preferrably one out of the first two to make it easier for the reviewers).
+
+
+### What are the requirements for the bounty hunter?
+
+- You know how to implement and deploy an API
+- You can implement an authentication-flow to allow signing up through a third-party account provider
+- You know how to store and retrieve simple lookups (fed-address/account-id relation)
+
+
+### What are the deliverables?
+
+- A server application that  
+  - allows configuring the domain, that is appended to the federation-addresses
+  - provides a simple but appealing UI where anyone can sign-up and login through a social-media account-provider of your choice
+    - allow a logged-in user to add, change or remove their stellar account-id
+    - presents the user with their full federation-address consisting of their username and the domain (e.g. `@tyvdh*quest.stellar.org`)
+    - allow logged-id user to delete their account
+  - serves the `/federation` endpoint that 
+    - supports at least the `name` query-type (see [federation protocol])
+- Documentation that 
```

Comment:
Should the location of the documentation be specified? Seems like a no-brainer that it would be alongside the application in the repo, but maybe it would be worth being verbose about?